src/app-layout/index.tsx

```diff
@@ -615,7 +615,7 @@ const OldAppLayout = React.forwardRef(
               ariaLabels={{
                 openLabel: activeDrawer?.ariaLabels?.triggerButton,
                 closeLabel: activeDrawer?.ariaLabels?.closeButton,
-                mainLabel: activeDrawer?.ariaLabels.drawerName,
+                mainLabel: activeDrawer?.ariaLabels?.drawerName,
```

Comment:
Expected source code change here. Caught an incompatibility between beta and public API.